src/components/ShowProgress.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import progressCircle from './ShowProgress.png';
+import './ShowProgress.css';
+
+const ShowProgress = ({ progress }) => {
+  const { currentChapter } = progress;
+
+  return (
+    <div className="book-progress">
+      <div className="book-progress-visualisation">
+        <img src={progressCircle} alt="Progress circle" className="circle" />
+        <div className="percentage">
+          <p className="display-per">65%</p>
+          <p className="text">Completed</p>
+        </div>
+      </div>
+      <div className="book-progress-update">
+        <div className="curent-chapter-wrapper">
+          <p className="text">CURRENT CHAPTER</p>
+          <p className="chapter">{currentChapter}</p>
+          <button type="button" className="update-button">
+            UPDATE PROGRESS
+          </button>
+        </div>
+      </div>
+    </div>
+  );
+};
+
```

Comment:
- [ ] Make this div to have same padding or margin currently they are not on the same line you can use `display: flex, align-items: center, justify-content: center`
![flex](https://user-images.githubusercontent.com/44745855/140499580-46c088bc-586b-4724-a3fb-2a9e75d0d70b.png)

Check the design yu will see what I mean

Reply:
[OPTIONAL] Good job! You added margin/padding top to your right progress card.

- [ ] but, it is still not identically centered as shown in the Zeplin and was requested by @DeliceLydia. I marked it as **optional**, but I highly recommend refactoring to center it a bit more;

Your book card:
![Screen Shot 2021-11-06 at 15 01 18](https://user-images.githubusercontent.com/60043698/140606260-6e7fff0a-1868-4fd3-81c6-a1e9916ddd2e.png)

Zeplin's:
<img width="921" alt="Screen Shot 2021-11-06 at 15 22 29" src="https://user-images.githubusercontent.com/60043698/140606297-e342cac6-3565-4cb9-8faf-abe8395004a8.png">